roles/keystone/tasks/main.yml

```diff
@@ -211,3 +211,12 @@
   tags:
     - serverspec
   when: serverspec.enabled|default('False')|bool
+
+- name: setup keystone inspec checks
+  blockinfile:
+    dest: "{{ inspec.control_path }}"
```

Comment:
maybe we should write these out into their own files and then have the control file include them ... either individually by lininfile a include line,  or as a `conf.d/*` style system.